app/components/forms/admin/settings/system/storage-option.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import { amazonS3Regions } from 'open-event-frontend/utils/dictionary/amazon-s3-regions';
+import { orderBy } from 'lodash';
+
+const { Component, computed } = Ember;
+
+export default Component.extend({
+  Regions: computed(function() {
+    return orderBy(amazonS3Regions);
+  }),
+
+  didInsertElement() {
```

Comment:
Please don't do it like this ... Use ember's attribute bindings on the input `disabled` property to do it. We need elements to be reactive to the data.